docs/source/chaincode4ade.rst

```diff
@@ -39,11 +39,17 @@ In each language, the ``Invoke`` method is called by clients to submit transacti
 proposals. This method allows you to use the chaincode to read and write data on
 the channel ledger.
 
-You also need to include an ``Init`` method that will serve as the initialization
-function for your chaincode. This method will be called in order to initialize
-the chaincode when it is started or upgraded. By default, this function is never
-executed. However, you can use the chaincode definition to request that the ``Init``
-function be executed. If execution of ``Init`` is requested, fabric will ensure
+If you use the low level APIs provided by the Fabric Chaincode Shim API,
+you need to include an ``Init`` method in your chaincode that will serve as the
+initialization function. This method needs to be called in order to initialize
+your chaincode when it is started or upgraded. By default, this function is never
```

Comment:
```suggestion
initialization function. You can use this method to implement logic required to initialize
your chaincode when it is started or upgraded. By default, this function is never
```

Reply:
I'm ok merging without this change, but I don't like the use of the word `needs`.  While you need to implement this to satisfy the interface, you can no-op it and it's not required to actually implement any real logic

Reply:
@mastersingh24 Good suggestion. Done.